rootfs/Dockerfile

```diff
@@ -9,21 +9,39 @@ RUN adduser --system \
 
 COPY requirements.txt /app/requirements.txt
 
-RUN buildDeps='gcc git libffi-dev libpq-dev python3-dev'; \
+RUN buildDeps='gcc git libffi-dev libpq-dev python3-dev python3-pip python3-wheel python3-setuptools'; \
     apt-get update && \
     apt-get install -y --no-install-recommends \
```

Comment:
I asked because I haven't found it in the e2e building phase but I have seen it in chef and such before. Depends if the `env` in ubuntu-slim isn't doing its job